### PR TITLE
docs: add info that license scanning supports file-patterns flag

### PR DESCRIPTION
**Pull request by DmitriyLewen**
## Description
License scanning with `license` analyzer supports [file-patterns](https://aquasecurity.github.io/trivy/v0.46/docs/configuration/skipping/#file-patterns) flag:

```bash
➜ ls -hl .
-rw-r--r--@ 1 work  staff    32K Nov  1 10:25 my-license-file

➜ trivy fs --scanners license --license-full .                 
2023-11-01T10:25:35.698+0600	INFO	Full license scanning is enabled

➜ trivy fs --scanners license --license-full --file-patterns "license-file:.*my-license-file" .
2023-11-01T10:27:02.501+0600	INFO	Full license scanning is enabled

Loose File License(s) (license)

Total: 1 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 1, CRITICAL: 0)

┌────────────────┬──────────┬─────────┬─────────────────┐
│ Classification │ Severity │ License │  File Location  │
├────────────────┼──────────┼─────────┼─────────────────┤
│ Restricted     │ HIGH     │ GPL-3.0 │ my-license-file │
└────────────────┴──────────┴─────────┴─────────────────┘
```

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [x] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [ ] I've added tests that prove my fix is effective or that my feature works.
- [ ] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [ ] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

File: docs/docs/configuration/skipping.md [hunk 1]
```diff
@@ -114,3 +114,5 @@ A file pattern contains the analyzer it is used for, and the pattern itself, joi
 ```
 
 The prefixes are listed [here](https://github.com/aquasecurity/trivy/tree/{{ git.commit }}/pkg/fanal/analyzer/const.go)
+
+[^1]: Only for licenses not obtained from packages (additional licenses obtained using the [license-full](../scanner/license.md) flag)
```
Comment (knqyf263) [inline docs/docs/configuration/skipping.md:118]: Can we simplify the footnote?
```suggestion

[^1]: Only work with the [license-full](../scanner/license.md) flag)
```

Comment (DmitriyLewen) [reply docs/docs/configuration/skipping.md:118]: I thought about this, but `license-full` flag includes licenses from packages.
Do you think this won't be confusing?

Comment (knqyf263) [reply docs/docs/configuration/skipping.md:118]: I think it is ok. Let's start with a simple explanation. If it confuses users, we can update it.

Comment (DmitriyLewen) [reply docs/docs/configuration/skipping.md]: Updated footnote in https://github.com/aquasecurity/trivy/pull/5484/commits/b56d24161fc5bc9349db05061bd70aa8014dae8a

Comment (knqyf263) [reply docs/docs/configuration/skipping.md]: Thanks!

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**